subt/ros/robot/launch/x4.launch

```diff
@@ -150,9 +150,12 @@
     <remap from="/points_cleaned" to="/$(arg robot_name)/rgbd_camera/depth/points_cleaned"/>
   </node>
 
+  <node name="map_tf_broadcaster" pkg="robot" type="map_tf_broadcaster.py" args="$(arg robot_name)">
+  </node>
+
   <node if="$(eval arg('robot_name').endswith('XM'))" ns="mapping" pkg="octomap_server" type="octomap_server_node" name="octomap">
     <remap from="cloud_in" to="/$(arg robot_name)/rgbd_camera/depth/points_cleaned"/>
-    <param name="frame_id" value="odom"/>
+    <param name="frame_id" value="map"/>
```

Comment:
Probably in a future PR, but we should do this in other nodes to and remove origin handling from cloudsim2osgar.

Reply:
I do not trust it yet ;) ... but it makes sense.